packages/cli/package.json

```diff
@@ -34,11 +34,13 @@
     "env-paths": "^3.0.0",
     "ora": "^7.0.1",
     "p-limit": "^4.0.0",
+    "prompts": "^2.4.2",
     "strip-indent": "^4.0.0",
     "yargs": "^17.7.2"
   },
   "devDependencies": {
     "@types/node": "^18.13.0",
+    "@types/prompts": "^2.4.4",
```

Comment:
that lib had a definition inside, they seem to have migrated the typedefs from def. typed to their package

Reply:
see https://github.com/terkelg/prompts

Reply:
idk whats the difference between those types, so just a hint

Reply:
My `zed` editor doesn't seem to recognise them. I will switch to `vscode` and check them 👍

Reply:
Looks like they are still not exposing the types in the same package when they publish to `npm`
https://www.npmjs.com/package/prompts?activeTab=code